src/main/java/org/sonar/plugins/csharp/RoslynProfileExporter.java

```diff
@@ -177,14 +185,78 @@ public void exportProfile(RulesProfile rulesProfile, Writer writer) {
     appendLine(writer, "</RoslynExportProfile>");
   }
 
-  public static ImmutableMultimap<String, ActiveRule> activeRoslynRulesByPartialRepoKey(List<ActiveRule> activeRules) {
-    ImmutableMultimap.Builder<String, ActiveRule> builder = ImmutableMultimap.builder();
+  private static String analysisSettings(boolean includeSettings, boolean ignoreHeaderComments, boolean includeRules, RulesProfile ruleProfile) {
+    StringBuilder sb = new StringBuilder();
+
+    appendLine(sb, "<?xml version=\"1.0\" encoding=\"UTF-8\"?>");
+    appendLine(sb, "<AnalysisInput>");
+
+    if (includeSettings) {
+      appendLine(sb, "  <Settings>");
+      appendLine(sb, "    <Setting>");
+      appendLine(sb, "      <Key>" + CSharpPlugin.IGNORE_HEADER_COMMENTS + "</Key>");
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Define a constant instead of duplicating this literal "</Key>" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1192)